api/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -82,6 +82,8 @@ type DatadogFeatures struct {
 	SBOM *SBOMFeatureConfig `json:"sbom,omitempty"`
 	// ServiceDiscovery
 	ServiceDiscovery *ServiceDiscoveryFeatureConfig `json:"serviceDiscovery,omitempty"`
+	// GPU monitoring
+	GPUMonitoring *GPUMonitoringFeatureConfig `json:"gpu,omitempty"`
```

Comment:
Sorry I missed this in the first pass.
```suggestion
	GPU *GPUFeatureConfig `json:"gpu,omitempty"`
```